netmap/attributes.go

```diff
@@ -113,19 +113,24 @@ func WriteSubnetInfo(node *NodeInfo, info NodeSubnetInfo) {
 // ErrRemoveSubnet is returned when a node needs to leave the subnet.
 var ErrRemoveSubnet = errors.New("remove subnet")
 
+var errNoSubnets = errors.New("no subnets")
+
```

Comment:
This looks like something outside of scope for the function named `IterateSubnets`. Could you provide a bit more context about where `IterateSubnets` is going to be used and what it is needed for? The current logic can be a part of the SDK with a different name.

Reply:
It is going to be used for two needs:

1. read-only iterating over all subnets (placement);
2. iterating over all subnets and removing some of them.